src/Files/Filesystem/ListedItem.cs

```diff
@@ -414,7 +414,7 @@ public override string ToString()
         public bool IsLinkItem => IsShortcutItem && ((ShortcutItem)this).IsUrl;
         public bool IsFtpItem => this is FtpItem;
         public bool IsZipItem => this is ZipItem;
-
+        public virtual bool IsImage => new[] { ".jpg", ".jpeg", ".bmp", ".png" }.Contains(Path.GetExtension(ItemPath), StringComparer.OrdinalIgnoreCase);
```

Comment:
I think we already have a property for this `selectedItemsPropertiesViewModel.IsSelectedItemImage`, is it possible to reuse it?

Reply:
I’ll have a look and check.

Reply:
I removed the IsImage property from this file, and made use of the SelectedItemsViewModel. I couldn't see anywhere I could access it without instantiating it, and using the CheckFileExtension method.

Reply:
It might make sense to create a helper for this

Reply:
I'll just double check the existing helpers and see if there was something already there, if not I'll find somewhere to add it.